packages/@aws-cdk/aws-ivs/lib/playback-key-pair.ts

```diff
@@ -0,0 +1,72 @@
+import * as core from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnPlaybackKeyPair } from './ivs.generated';
+
+/**
+ * Represents an IVS Playback Key Pair
+ */
+export interface IPlaybackKeyPair extends core.IResource {
+  /**
+   * Key-pair ARN. For example: arn:aws:ivs:us-west-2:693991300569:playback-key/f99cde61-c2b0-4df3-8941-ca7d38acca1a
+   *
+   * @attribute
+   */
+  readonly playbackKeyPairArn: string;
+}
+
+/**
+ * Reference to a new or existing IVS Playback Key Pair
+ */
+abstract class PlaybackKeyPairBase extends core.Resource implements IPlaybackKeyPair {
+  // these stay abstract at this level
+  public abstract readonly playbackKeyPairArn: string;
+}
+
+/**
+ * Properties for creating a new Playback Key Pair
+ */
+export interface PlaybackKeyPairProps {
+  /**
+   * The public portion of a customer-generated key pair.
+   */
+  readonly publicKeyMaterial: string;
+
+  /**
+   * An arbitrary string (a nickname) assigned to a playback key pair that helps the customer identify that resource.
+   * The value does not need to be unique.
+   *
+   * @default Empty String
```

Comment:
```suggestion
   * @default ''
```

String defaults are surrounded in single quotes, if it's truly an empty string (and not undefined) it should be `''`, otherwise it should be `- None`